apps/crud/src/app/store/todo.store.ts

```diff
@@ -0,0 +1,50 @@
+import { ComponentStore } from '@ngrx/component-store';
+import { TodoConfig } from '../core/Interface/todo';
+import { Injectable } from '@angular/core';
+interface ToDOState {
+  todos: TodoConfig[] | [];
+  localSpinnerState: boolean;
+  globalSpinnerState: boolean;
+}
+
+const initialState: ToDOState = {
+  todos: [],
+  localSpinnerState: false,
+  globalSpinnerState: false,
+};
+
+@Injectable({
+  providedIn: 'root',
```

Comment:
this service is linked to the life cycle of a component, so it shouldn't be injeted into root but provided at component level

Reply:
done